Lively/src/components/session/TeamEntryForm.tsx

```diff
@@ -0,0 +1,166 @@
+import * as React from 'react';
+import './TeamEntry.css';
+import { handleTokens } from '../../util';
+import { Submit, Loading } from '../wrappers/Buttons';
+import { validateEmail } from '../../util';
+import { MutationFunc } from 'react-apollo';
+
+interface State {
```

Comment:
more repeated stuff. some of this seems strikingly similar to other interfaces that exist elsewhere